chore(design-system): bundle css & externalize vue (#611)
opencloud-eu/web: packages/design-system/README.md

## Usage as a package

### Installation

To use the design-system in your application, you first need to install the package. Depending on your package manager, run one of the following commands:

```
$ npm install @opencloud-eu/design-system

> $ pnpm add -D @opencloud-eu/design-system

Reviewer: ```suggestion
$ pnpm add @opencloud-eu/design-system
```

$ yarn add @opencloud-eu/design-system
```

### Styles

In order to use the provided CSS classes and to ensure the components are styled correctly, you need to import the styles like so:

```
import '@opencloud-eu/design-system/dist/design-system.css'
```

### Components

> To use the components, you need to import the component you want to use. For example, to use the `OcButton` component:

Reviewer: Is it possible to register all components globally?

## Usage as a package

### Installation

To use the design-system in your application, you first need to install the package. Depending on your package manager, run one of the following commands:

```
$ npm install @opencloud-eu/design-system

$ pnpm add @opencloud-eu/design-system

$ yarn add @opencloud-eu/design-system
```

Note that if you're using the design-system in an OpenCloud Web app, it's recommended to install it as dev dependency. This is because the Web runtime already ships the design-system and you only need it for development purposes in your IDE.

### Styles

In order to use the provided CSS classes and to ensure the components are styled correctly, you need to import the styles like so:

```
import '@opencloud-eu/design-system/dist/design-system.css'
```

Again, this is not needed if you're using the design-system in an OpenCloud Web app because the styles are already available via the Web runtime.

### Components

> To use the components, you need to import the component you want to use. For example, to use the `OcButton` component:

Reviewer: ```suggestion
To use a component, you need to import it. For example, to use the `OcButton` component:
```
